### ISSUE TITLE
Logging flooded by error messages

### ISSUE THREAD

**Issue by beloware** [MonkeyTedBaker/Monocle#22]
Logfile flooded error messages like:

[2017-09-16 11:37:04][ WARNING][hashing] Error 429 - Out of hashes for this period.

Making log useless for any error search.

**Comment by MonkeyTedBaker** [comment]
Umm.. upgrade hashing? xD
Or use grep to search?

**Comment by beloware** [comment]
Nope,  only restarting instance causes the problem. Key is enough for normal operation.
Grep not worth anything when all log is rolling over in short time. 
10k lines in log is average when it occures.

**Comment by captainmedic** [comment]
Though my PR does not remove the hashing error, it does allow you to store more logs in case you need it, all you do is edit one line in the config file.

**Comment by MonkeyTedBaker** [comment]
Can you pls close this issue if unnecessary?
